frontend/.eslintrc.js

```diff
@@ -9,19 +9,29 @@ module.exports = {
     }
   },
   plugins: ["ember"],
-  extends: ["eslint:recommended", "plugin:ember/recommended"],
+  extends: [
+    "eslint:recommended",
+    "plugin:ember/recommended",
+    "plugin:prettier/recommended"
+  ],
   env: {
     browser: true
   },
   rules: {
     "ember/no-global-jquery": "off",
+    "ember/no-classic-classes": "off",
+    "ember/require-tagless-components": "off",
+    "ember/no-classic-components": "off",
+    "ember/no-component-lifecycle-hooks": "off",
+    "space-before-function-paren": ["off", "always"],
```

Comment:
maybe double-check those rules and look up if they make sense to you

Reply:
Those rules are needed for the imports of glimmer and classic classes. I've set them to off, because it's a lot of work to change those import in every file. 
For now though,  I think we are fine if we leave it as it is.